pkgs/servers/owncast/default.nix

```diff
@@ -25,7 +27,7 @@ in buildGoModule {
 
   postInstall = ''
     wrapProgram $out/bin/owncast \
-      --prefix PATH : ${lib.makeBinPath [ ffmpeg ]}
+      --prefix PATH : ${lib.makeBinPath [ bash which ffmpeg ]}
```

Comment:
We could instead just patch https://github.com/owncast/owncast/blob/66fa8866f6b7f25d7c8da0f0d20c564d463626cb/utils/utils.go#L225C6-L225C25 to return the absolute path of your ffmpeg